### PR TITLE
refactor(interactive): flex storage with mmap array

### PR DESCRIPTION
**Pull request by liulx20**
refactor flex storage with mmap array

## Related issue number

Fixes

### CODE REVIEW

File: flex/bin/CMakeLists.txt [hunk 1]
```diff
@@ -56,4 +49,14 @@ if(BUILD_HQPS)
         endif()
         # install the script
         install(PROGRAMS load_plan_and_gen.sh DESTINATION bin)
-endif()
\ No newline at end of file
+endif()
+
+find_package(Boost COMPONENTS program_options REQUIRED)
```
Comment (zhanglei1949) [inline flex/bin/CMakeLists.txt:54]: Dependency to `boost program_options` has already been added in flex/CMakeLists.txt, no need to find again.

**Comment by longbinlai** [comment]
can we add an interactive doc to introduce how the mmap can help offloading graph data so that we can support larger graph.

File: flex/storages/rt_mutable_graph/loader/basic_fragment_loader.cc [hunk 2]
```diff
@@ -36,28 +43,58 @@ void BasicFragmentLoader::init_vertex_data() {
...
-  res_fragment.vertex_data_.swap(vertex_data_);
-  res_fragment.lf_indexers_.swap(lf_indexers_);
-  VLOG(10) << "Finish Building Fragment, " << res_fragment.vertex_label_num_
-           << " vertices labels, " << res_fragment.edge_label_num_
-           << " edges labels";
+  for (label_t v_label = 0; v_label < vertex_label_num_; v_label++) {
+    auto& v_data = vertex_data_[v_label];
+
+    auto label_name = schema_.get_vertex_label_name(v_label);
+    // lf_indexers_[v_label].dump_keys(vertex_map_prefix(label_name) + ".keys",
```
Comment (zhanglei1949) [inline flex/storages/rt_mutable_graph/loader/basic_fragment_loader.cc:66]: Can be removed?

File: flex/utils/allocators.h [hunk 3]
```diff
@@ -88,6 +91,135 @@ class ArenaAllocator {
       typed_allocations_;
 };
 
+#if 0
```
Comment (zhanglei1949) [inline flex/utils/allocators.h:94]: Can be removed?

**Review by zhanglei1949 — DISMISSED** [review]
LGTM

**Review by luoxiaojian — APPROVED** [review]

**Review by zhanglei1949 — APPROVED** [review]
LGTM

**Pull request merged**